docs/editor/accessibility.md

```diff
@@ -25,12 +25,76 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Customizing warning colors
+
+The default Color Theme for VS Code is **Dark+**. However, you can customize both the theme and property colors in the user interface.
+
+> Note: Visit [Customizing a Color Theme](/docs/getstarted/themes.md#customizing-a-color-theme) to learn more about overriding the colors in your current theme.
+
+To customize the error/warning squigglies, go to **File** > **Preference** > **Settings** (**Code** > **Preference** > **Settings** for macOS). Then open the UI settings by selecting the file icon shown in the image below.
```

Comment:
I think you want to direct the user to open their user settings.json file by searching for "color customizations" then selecting the "Edit in settings.json" link.
I'm not sure what you mean by "selecting the file icon shown in the image below" since it is cropped out of the image.